rclcpp/include/rclcpp/callback_group.hpp

```diff
@@ -49,77 +49,46 @@ class CallbackGroup
 public:
   RCLCPP_SMART_PTR_DEFINITIONS(CallbackGroup);
 
-  CallbackGroup(CallbackGroupType group_type)
-  : type_(group_type), can_be_taken_from_(true)
-  {}
-
-  const std::vector<subscription::SubscriptionBase::WeakPtr> &
-  get_subscription_ptrs() const
-  {
-    return subscription_ptrs_;
-  }
-
-  const std::vector<timer::TimerBase::WeakPtr> &
-  get_timer_ptrs() const
-  {
-    return timer_ptrs_;
-  }
-
-  const std::vector<service::ServiceBase::SharedPtr> &
-  get_service_ptrs() const
-  {
-    return service_ptrs_;
-  }
-
-  const std::vector<client::ClientBase::SharedPtr> &
-  get_client_ptrs() const
-  {
-    return client_ptrs_;
-  }
-
-  std::atomic_bool & can_be_taken_from()
-  {
-    return can_be_taken_from_;
-  }
-
-  const CallbackGroupType & type() const
-  {
-    return type_;
-  }
+  CallbackGroup(CallbackGroupType group_type);
+
+  const std::vector<rclcpp::subscription::SubscriptionBase::WeakPtr> &
```

Comment:
I think the headers should us the fully qualified names so people reading it can easily track down what the full namespace of parameters are. Plus I had to do this in the .cpp file or do `using namespace rclcpp` which I was trying to avoid and this made the function prototypes in the hpp and cpp files consistent with each other.